models/yang/sonic/sonic-fdb.yang

```diff
@@ -15,11 +15,33 @@ module sonic-fdb {
         description
                 "SONIC FDB";
 
+        revision 2019-11-07 {
```

Comment:
Same comment about keeping indentations at 4 spaces. Fix it throughout the file if you need to.